docs/apis/websocket.md

```
@@ -0,0 +1,96 @@
+  ],
+  "notUnregistered": {
+    "calendars/userC/99999" : "internal error"
+  }
+}
+```
+
+#### Calendar Event push
+
+Whenever a change happens on a registered calendar (new event, update, deletion, sync-token update), the side-service
+pushes an event message to the client.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)